function/ÜbungenzuFunktionen1.js

```diff
@@ -0,0 +1,21 @@
+const readline = require('readline');
+const rl = readline.createInterface({input: process.stdin, output: process.stdout});
+const prompt = (query) => new Promise((resolve) => rl.question(query, resolve));
+rl.on('close', () => process.exit(0));
+
+async function execute() { // put all your codes here for user inputs
+
```

Comment:
In diesen Funktionen brauchst du keine execute function, weil du keinen "asynchronen Kontext" hast, in anderen Worten, du wartest nicht auf die Eingabe eines Users, oder ähnliches

Reply:
Das gilt auch für die anderen Aufgaben ;)

Reply:
The "asynchronen" function is removed for all excercises.